### PR TITLE
feat(applications-service): add feature toggle

### PR DESCRIPTION
**Pull request by jonathangoulding**
toggle: feature_enabled_content_security_policy

### CODE REVIEW

**Review by DavidBeale — APPROVED** [review]

**Review by Philip-Philippou1 — COMMENTED** [review]
I think you need to update the Readme
app/stacks/uk-west/applications-service/README.md

app/stacks/uk-south/applications-service/README.md

File: config/variables/stacks/applications-service/test.hcl [hunk 1]
```diff
@@ -7,6 +7,7 @@ locals {
   feature_redis_session_store                                     = "true"
   feature_save_and_exit_option                                    = "false"
   feature_show_affected_area_section                              = "false"
+  feature_enabled_content_security_policy                         = "false"
```
Comment (Philip-Philippou1) [inline config/variables/stacks/applications-service/test.hcl:10]: Any reason to have this off in test?

**Comment by jonathangoulding** [comment]
> I think you need to update the Readme app/stacks/uk-west/applications-service/README.md
> 
> app/stacks/uk-south/applications-service/README.md

done

File: config/variables/stacks/applications-service/test.hcl (hunk 1 above)
Comment (jonathangoulding) [reply config/variables/stacks/applications-service/test.hcl:10]: nope will put it on

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**